### DISCUSSION TITLE
Sprint 1 daily

### DISCUSSION THREAD

**Discussion by YaakovHatam** [YaakovHatam/diet-recipes#32] [General]
# Date
what i did yesteryday
what i doing today

**Comment by ShlomoRussell** [comment]
Sun 27/3/2022
I finished the search functions and I am open to do more issues or to help others if needed.

**Comment by Amicht** [comment]
I finished the random recipe component, 
just waiting for the logic function and I will update it.

**Comment by OsnatDayan** [comment]
כמעט סיימתי את המסך של העגלה יש לי רק בעיה קטנה במחיקה שאני מקווה לסיים מחר בעז"ה

**Comment by OsnatDayan** [comment]
סיימתי את המסך של העגלה 28.03
